crates/l2/proposer/prover_server.rs

```diff
@@ -41,7 +41,9 @@ pub async fn start_prover_server(store: Store) {
     let eth_config = EthConfig::from_env().expect("EthConfig::from_env()");
     let proposer_config = CommitterConfig::from_env().expect("CommitterConfig::from_env()");
     let mut prover_server =
-        ProverServer::new_from_config(server_config.clone(), &proposer_config, eth_config, store);
+        ProverServer::new_from_config(server_config.clone(), &proposer_config, eth_config, store)
+            .await
+            .unwrap(); // Using an unwrap, if we can't initialize the prover_server any error_handling is pointless, we may want to retry inside the initialization.
```

Comment:
If error handling is pointless and unwrapping makes sense, use `expect` instead to add context. This applies to every use of `unwrap`.

Reply:
Sure, thank you!